browser-test/src/support/admin_programs.ts

```diff
@@ -87,6 +87,13 @@ export enum ProgramLifecycle {
   ACTIVE = 'Active',
 }
 
+export enum ProgramAction {
```

Comment:
Another option was combining the "extra actions" and "actions" in one enum. However, I think it makes it more clear to have them separate so in the test we know which ones appear on the card vs on the dropdown

I did combine the `expect()` method into one that takes both action and extra action because it makes the test easier to digest. WDYT?